steamcmd/logs/configstore_log.txt


[2022-07-09 11:15:43] Failed to read store 'install' from 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf.tmp'
[2022-07-09 11:15:43] Failed to read store 'install' from 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf.tmp'
[2022-07-09 11:15:43] Failed to read store 'install' from 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf.tmp'
[2022-07-09 11:15:44] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'


[2022-07-09 11:20:00] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 11:20:30] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 11:21:00] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 11:21:30] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 11:22:01] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 11:22:31] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 11:23:01] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 11:23:31] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 11:24:01] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 11:24:27] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\userdata\anonymous\config\localconfig.vdf'
[2022-07-09 11:24:31] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 11:25:01] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 11:25:31] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 11:26:01] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 11:26:31] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 11:27:01] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'


[2022-07-09 19:30:15] Loaded store 'install' from 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf' successfully
[2022-07-09 19:30:15] Loaded store 'install' from 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf' successfully
[2022-07-09 19:30:15] Loaded store 'install' from 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf' successfully
[2022-07-09 19:30:21] Loaded store 'install' from 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf' successfully
[2022-07-09 19:30:23] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 19:30:38] Failed to read store 'userlocal' from 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\userdata\206729962\config\localconfig.vdf.tmp'
[2022-07-09 19:30:38] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 19:30:39] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\userdata\206729962\config\localconfig.vdf'
[2022-07-09 19:30:39] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\userdata\206729962\config\localconfig.vdf'
[2022-07-09 19:30:39] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 19:30:40] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\userdata\206729962\config\localconfig.vdf'
[2022-07-09 19:30:44] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\userdata\206729962\config\localconfig.vdf'
[2022-07-09 19:30:44] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\userdata\206729962\config\localconfig.vdf'
[2022-07-09 19:31:18] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\userdata\206729962\config\localconfig.vdf'
[2022-07-09 19:31:19] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 19:31:19] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 19:31:20] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\userdata\206729962\config\localconfig.vdf'
[2022-07-09 19:31:22] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\userdata\206729962\config\localconfig.vdf'
[2022-07-09 19:31:50] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 19:32:20] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 19:32:50] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 19:33:20] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 19:33:50] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 19:34:20] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 19:34:50] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 19:35:20] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 19:35:50] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 19:36:20] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 19:36:50] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 19:37:20] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 19:37:50] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 19:38:00] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\userdata\206729962\config\localconfig.vdf'
[2022-07-09 19:38:20] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 19:38:50] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 19:39:21] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'
[2022-07-09 19:44:13] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\userdata\206729962\config\localconfig.vdf'
[2022-07-09 19:59:26] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\userdata\206729962\config\localconfig.vdf'
[2022-07-09 20:15:56] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\userdata\206729962\config\localconfig.vdf'
[2022-07-09 20:30:40] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\userdata\206729962\config\localconfig.vdf'


[2022-07-27 00:45:15] Loaded store 'install' from 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\config\config.vdf' successfully
[2022-07-27 00:45:15] Loaded store 'install' from 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\config\config.vdf' successfully
[2022-07-27 00:45:15] Loaded store 'install' from 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\config\config.vdf' successfully
[2022-07-27 00:45:22] Loaded store 'install' from 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\config\config.vdf' successfully
[2022-07-27 00:45:24] Loaded store 'userlocal' from 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf' successfully
[2022-07-27 00:45:24] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\config\config.vdf'
[2022-07-27 00:45:24] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 00:45:27] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 00:45:29] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 00:45:59] Loaded store 'install' from 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\config\config.vdf' successfully
[2022-07-27 00:46:00] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\config\config.vdf'
[2022-07-27 00:46:30] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\config\config.vdf'
[2022-07-27 01:01:25] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 01:15:26] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 01:30:06] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 01:46:13] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 02:01:13] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 02:14:53] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 02:29:29] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 02:45:36] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 03:00:25] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 03:14:13] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 03:29:40] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 03:45:26] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 04:00:28] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 04:14:26] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 04:29:36] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 04:44:25] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 04:59:01] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 05:13:11] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 05:27:49] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 05:42:49] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 05:58:21] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 06:14:36] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 06:30:14] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 06:44:00] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 06:59:37] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 07:15:39] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 07:30:08] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 07:43:50] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 07:59:44] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 08:15:30] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 08:30:07] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 08:45:58] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 08:59:37] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 09:02:12] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\config\config.vdf'
[2022-07-27 09:02:33] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\config\config.vdf'
[2022-07-27 09:02:45] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\config\config.vdf'
[2022-07-27 09:03:15] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\config\config.vdf'
[2022-07-27 09:03:15] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\config\config.vdf'
[2022-07-27 09:04:40] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\config\config.vdf'
[2022-07-27 09:04:43] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\config\config.vdf'
[2022-07-27 09:04:43] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 09:04:44] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 09:50:33] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 10:06:34] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 10:20:24] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 10:36:06] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 10:51:27] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 11:06:03] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 11:21:51] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'
[2022-07-27 11:35:48] Flushed store 'userlocal' to 'C:\Users\kazza\Desktop\Garry's Mod\steamcmd\userdata\anonymous\config\localconfig.vdf'


Garry's Mod/logs/configstore_log.txt


[2022-07-09 11:15:43] Failed to read store 'install' from 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf.tmp'
[2022-07-09 11:15:43] Failed to read store 'install' from 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf.tmp'
[2022-07-09 11:15:43] Failed to read store 'install' from 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf.tmp'
[2022-07-09 11:15:44] Flushed store 'install' to 'C:\Users\kazza\Desktop\Garry's Mod\Garry's Mod\config\config.vdf'


steamcmd/steamapps/common/GarrysModDS/garrysmod/addons/usepickup/lua/usepickup/config.lua
local c = {}

c.Enable        = true
c.LoadGM        = "auto" -- "auto", "terrortown", "darkrp", "fallback"
                     -- IF YOUR GAMEMODE IS JUST DARKRP BUT RENAMED, SET IT TO "darkrp" (if it doesn't detect DarkRP by itself)

c.Language      = "english" -- english, german, russian

c.Range         = 125 -- 125 => perfect in my opinion
c.Halos         = true
c.AutoPickup    = true

c.ChatCommands = {
    "!usepickup",
    "/usepickup",
    ".usepickup"
}

c.Colors = {
    ["Base"]    = Color( 220, 220, 220 ), -- normal text
    ["UseKey"]  = Color( 200, 50, 50 ), -- [E]
    ["Alt"]     = Color( 140, 140, 140 ), -- "Hold [ALT][..]" text
    ["Shadow"]  = Color( 50, 50, 50 ), -- shadow

    ["Row_BG"]  = Color( 30, 30, 30, 200 ), -- blurry background color of each row
    ["Row_Bar"] = Color( 150, 150, 150),
    ["Good"]    = Color( 0, 150, 0 ),
    ["Bad"]     = Color( 175, 0, 0 ),
    ["Avg"]     = Color( 255, 215, 0 ),

    ["SettingsBG"] = Color( 25, 25, 25, 220 ),
    ["SettingsCloseButton"] = Color( 200, 35, 35 )
}
c.NameOverride = { -- if for some reason the name doesnt fit or is just fucked up
    ["weapon_zm_revolver"] = "Desert Eagle",
}
c.ColorOverride = { -- generates the weapons color by its ammotype (non ttt gamemodes) but you can override that here
    ["ammotype"] = Color( 200, 30, 30 ),
}

c.StatsBaseBlacklist = {
    "weapon_base",
    "weapon_tttbasegrenade"
}

c.StatsClassBlacklist = { -- if it doesnt detect those weapons itself (mostly equipment like t-items with 0 delay etc)
    "weapon_crowbar"
}

if CLIENT then
    -- play around
    -- I will add better names for the font later

    surface.CreateFont( "USEPICKUP.Standard", {
        font = "Coolvetica",
        extended = false,
        size = 24,
        weight = 0,
        antialias = true,
        shadow = false
    } )

    surface.CreateFont( "USEPICKUP.Highlighted", {
        font = "Coolvetica",
        extended = false,
        size = 24,
        weight = 0,
        antialias = true,
        shadow = false
    } )

    surface.CreateFont( "USEPICKUP.HoldAlt", {
        font = "Trebuchet MS",
        size = 21,
        weight = 1000,
        shadow = false,
        antialias = true,
    } )

    surface.CreateFont( "USEPICKUP.Stats.Standard", {
        font = "Coolvetica",
        extended = false,
        size = 24,
        weight = 0,
        antialias = true,
        shadow = false
    } )

    surface.CreateFont( "USEPICKUP.Stats.Highlighted", {
        font = "Coolvetica",
        extended = false,
        size = 26,
        weight = 1000,
        antialias = true,
        shadow = false
    } )

    surface.CreateFont( "USEPICKUP.Stats.Standard.Alt", {
        font = "Trebuchet MS",
        extended = false,
        size = 19,
        weight = 0,
        antialias = true,
        shadow = false
    } )
end

--                                                                                                                                                                                                                                                                                                                                                                                                   76561207038545733

USEPICKUP.Config = c